lib/cisco_node_utils/bgp_af.rb

```diff
@@ -398,9 +422,15 @@ def dampening=(damp_array)
         CiscoLogger.debug("Dampening 'dampening #{damp_array.join(' ')}''")
       elsif route_map.is_a? String
```

Comment:
Actually this is the part that confused me with respect to the below code - @RichWellum we're setting `route_map = ''` above and not touching it again until we get here, so this `elsif route_map.is_a? String` will always pass - right? So really this is just an `else` in effect.